packages/babel-node/test/.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Ok, this is strange to me because the you don't need an empty config. The test may require it.